Documents/Management/WeeklyReports/weeklyReportFifthWeek.md

```diff
@@ -0,0 +1,36 @@
+# FifthWeeklyReport
+
+[Cumulative Weekly Reports](mainWeeklyReport.md)
+
+## Week 5 (8-10 April 2024)
+
+The fifth week was a shorter week due to the Hackathon. The team finished working with LEDs. The team also finished working on the test plan, the user manual, and the code documentation.
+
+## This week's topics
+
+1. Finish to Manage each component with LEDs.
```

Comment:
Same as in the previous document, why there is a M in uppercase in the middle of the sentence?

Reply:
fixed